python/orca/src/bigdl/orca/learn/pytorch/pytorch_ray_estimator.py

```diff
@@ -330,8 +333,22 @@ def predict(self,
                 data = process_xshards_of_pandas_dataframe(data, feature_cols)
             pred_shards = self._predict_spark_xshards(data, param)
             result = update_predict_xshards(data, pred_shards)
+        elif isinstance(data, ray.data.Dataset):
+            shards = data.split(n=self.num_workers, locality_hints=self.remote_workers)
+
+            data_creator = lambda config, batch_size: shard
```

Comment:
Shall we also add the `to_torch` part in `data_creator`, as did in `fit` and `evaluate`? In that case, there might be fewer modifications to `pytorch_ray_worker` and `torch_runner`

Reply:
Solved.